requirements.txt

```diff
@@ -483,7 +483,8 @@ wcwidth==0.2.5
     # via
     #   prettytable
     #   prompt-toolkit
-websockets==7.0
+websockets==7.0 ; python_version<"3.9"
+websockets==8.0 ; python_version>="3.9"
```

Comment:
Similar fix as in [libjuju](https://github.com/juju/python-libjuju/blob/master/setup.py#L38-L39)